src/components/Home/StationModalContent.tsx

```diff
@@ -6,238 +6,172 @@ import BaseWeatherIcon from "@/components/BaseComponents/BaseWeatherIcon";
-                                        <span className="block font-bold text-xs">
-                                            {elem.barometer} hPa
-                                        </span>
-                                    </p>
-                                </div>
-                            </div>
-                            <div className="border-[0.5px] border-white h-[1px] my-4"></div>
-                            <div className="flex items-center justify-around">
-                                <div className="flex items-center w-[100px]">
-                                    <div className="h-4 w-6">
-                                        <SvgInline
-                                            path="weather_icons/wind.svg"
-                                            title="Wind icon"
-                                            className="fill-white"
-                                            style={{
-                                                transform: `rotate(${elem.winddir}deg)`,
-                                            }}
-                                        />
-                                    </div>
-                                    <p className="text-sm text-white leading-tight ml-2">
-                                        Wind
-                                        <span className="block font-bold text-xs">
-                                            {elem.windspd} km/h
-                                        </span>
-                                    </p>
-                                </div>
-                                <div className="flex items-center w-[100px]">
-                                    <div className="h-6 w-6">
-                                        <SvgInline
-                                            path="weather_icons/rain.svg"
-                                            title="Rain icon"
-                                            className="fill-white"
-                                        />
-                                    </div>
-                                    <p className="text-sm text-white leading-tight ml-2">
-                                        Rain
-                                        <span className="block font-bold text-xs">
-                                            {elem.percipitation} mm
-                                        </span>
-                                    </p>
-                                </div>
-                            </div>
+    return (
+        <div>
+            {weatherData.map((elem: ExportedWeatherData) => {
+                return (
+                    <div className="p-2 flex text-black flex-col relative" key={elem.station.id}>
+                        {isLoading && loadingBlock}
+                        <div className="flex flex-col">
+                            <StationWeatherSummary {...elem} />
+                            <StationWeatherDetails {...elem} />
```

Comment:
I like this so very much. Much much cleaner approach than before and probably we could move the `StationWeatherSummary` and `StationWeatherDetails` in different file components as well in the future.

Reply:
Brilliant, and yep I agree, I'm happy to move them out now if you agree.